app/models/facility.rb

```diff
@@ -101,8 +102,20 @@ def to_s
     "#{name} (#{abbreviation})"
   end
 
+  def problem_order_details
+    complete_problem_order_details.non_reservations
```

Comment:
And then I was thinking `complete_problem_order_details` would be ALL problem order details.